services/slam/builtin/cartographer_int_test.go

```diff
@@ -222,7 +222,7 @@ func integrationtestHelperCartographer(t *testing.T, mode slam.Mode) {
 		}
 	}
 
-	testCartographerPosition(t, svc)
+	testCartographerPosition(t, svc, "")
```

Comment:
[minor] Can we please comment on why the sensor is an empty string here?

Reply:
[q] Also, why does orbslam not have a sensor of `""` case?

Reply:
added a comment to address this, but in offline mode cartographer does not have information on the component reference, and our slam config leaves `sensors` empty as it is not required for offline mode currently. orbslam is different because it interprets the component reference from the config file. I also updated the orbslam integration test to make this clear

Reply:
Makes sense. Thanks!

Reply:
Additional note: This behavior will undergo changes in full modularization once we have the replay sensors